apis/kueue/v1beta1/workload_types.go

```diff
@@ -71,6 +71,12 @@ type PodSetFlavors struct {
 
 	// Flavors are the flavors assigned to the workload for each resource.
 	Flavors map[corev1.ResourceName]ResourceFlavorReference `json:"flavors,omitempty"`
```

Comment:
we are on time to change `PodSetFlavors` by something else, given that it's no longer just about flavors.

Maybe `PodSetAssignments` is a bit more generic?

And we could have a single list holding both the quantity and the flavor, instead of two maps. I'm not to sure about this one. How does the calculation of  `workload.Info` change if we do this?

Reply:
For the first part done.

For the second one ... , it will imply adding two addition conversion methods ... but it will look strange. Also since ResourceList is "standard", I'd say to keep it as is. (also the time that will take to re-re-work the tests might not be negligible )